Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM python:2.7.8-wheezy
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+RUN curl http://python-distribute.org/distribute_setup.py | `which python`
+RUN curl https://raw.github.com/pypa/pip/master/contrib/get-pip.py | `which python`
+
+RUN apt-get update -qq
+RUN apt-get install unzip -y
+RUN apt-get install fish -y
+RUN apt-get install mktemp
+RUN apt-get install -y build-essential\
+    git-core\
+    libldap2-dev\
+    libpq-dev\
+    libsasl2-dev\
+    libssl-dev\
+    libxml2-dev\
+    libxslt1-dev\
+    libffi-dev\
+    python-dev\
+    python-setuptools\
+    zlib1g-dev\
+    postgresql-client
+
+RUN pip install harvest
```

Comment:
Move `harvest` to the `requirements.txt` file in this repo and remove this line.